libs/@guardian/source-react-components/src/link/styles.ts

```diff
@@ -13,26 +14,28 @@ import type { IconSide, LinkPriority } from './@types/SharedLinkProps';
 import type { ThemeLink } from './theme';
 import { themeLink as defaultTheme } from './theme';
 
-export const link = css`
-	position: relative;
-	${textSans.medium()};
-	cursor: pointer;
-	text-decoration: underline;
-	text-underline-position: under;
-	text-underline-offset: 5%;
+export const link = (typographyPreset = textSans17) => {
+	return css`
+		position: relative;
+		cursor: pointer;
+		text-decoration: underline;
+		text-underline-position: under;
+		text-underline-offset: 5%;
+		${typographyPreset};
+		display: inline;
+		text-decoration-thickness: 0.1em;
 
```

Comment:
Could we nudge this up a couple of lines so it's grouped with the other text decoration properties?